source/mainnet/net/guides/create-account.rst

```diff
@@ -88,6 +88,10 @@ To learn more about identities and accounts, see :ref:`Identities <reference-id-
 
     .. tab:: |mw-gen1|
 
+        .. Note::
```

Comment:
Does it make sense to still have the guide below, if it can't really be used for anything?

Reply:
Removed